core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiService.scala

```diff
@@ -55,18 +55,7 @@ trait BillingApiService extends UserInfoDirectives {
                 }
               }
           }
-      } ~
-        path("billing") {
-          post {
```

Comment:
Is there an orchestration passthrough that needs to be removed?

Reply:
Yes, PR coming soon